irods/test/setupssl.py

```diff
@@ -73,7 +73,20 @@ def test(options, args=()):
     
 if __name__ == '__main__':
     import getopt
-    opt, arg_list = getopt.getopt(sys.argv[1:],'x:fh:kq')
+    try:
+        opt, arg_list = getopt.getopt(sys.argv[1:],'x:fh:kq')
+    except getopt.GetoptError:
+        print("""Usage: {sys.argv[0]} [-f] [-h <hostname>] [-k] [-q] [-x <extension>] 
+        -f  Force replacement of the existing SSL directory (/etc/irods/ssl) with a new one, containing newly generated files.
+        -h  In the generated certificate, use the given hostname rather than the value returned from socket.gethostname()
```

Comment:
Consider reserving `-h` for printing the help text. Or add a few lines to the README explaining how to view the help text.

Reply:
There's a README in the test directory too in. Either one would be fine IMO.

Reply:
Yeah, let's not change the option now. I think some explanatory text is fine.

I would argue that we should add it to the README - possibly in a separate PR - for expediency.

Reply:
I'm proposing that what you have now is fine and we can add to the README (and the help text itself, *later*) instructions for viewing help text. We can also do it now - whatever works.

Reply:
Leave everything as is for and open an issue to add additional information about using the script. The issue can reference this entire conversation.

Reply:
> Leave everything as is for now and open an issue to add additional information about using the script. The issue can reference this entire conversation.

Done